feat: :sparkles: Servidor creado con NodeJs y Express como tarea solicitada

CaroSerrano opened the pull request:
> [!IMPORTANT]
> Checklist a modo recordatorio. Por favor, describe brevemente los cambios que realizaste. ¿Qué tareas resuelve este PR?

## Lista de control(Verificar la lista de control antes del PR):
- [x] He realizado pruebas locales de mis cambios.
- [x] He comentado mi código, en particular en las áreas difíciles de entender.
- [x] He hecho los cambios necesarios en la documentación.
- [x] Mis cambios no generan nuevas advertencias.

## Otros comentarios
Este pull request genera un servidor web local a través de NodeJs y utilizando el framework Express. Está configurado para iniciarse mediante de Nodemon. Se generaron las rutas raiz, cursos y contacto. Además, utilizando middlewears se creó una ruta para manejar las inexistentes.

CaroSerrano committed:
feat: :sparkles: Servidor creado con NodeJs y Express como tarea solicitada

CaroSerrano commented:
Perdón por la tardanza. Recién hoy vi que lo pedían para antes del sábado. En la semana ando media complicada.

mariaelisaaraya commented:
Hola Caro! ¿Qué número de tarea realizaste? Entiendo que es la última tarea por lo que leo pero necesito que aclares que tarea realizas en el commit o en la conversation.
¡Gracias!

CaroSerrano commented:
Holaa! si, es la última. También me di cuenta de que en el apuro los archivos los puse en la carpeta principal y no armé una carpeta aparte. Eso lo pueden cambiar ustedes? o lo tengo que hacer en otro PR?

mariaelisaaraya commented:
Hola Caro,

¡Gracias por tu respuesta! Entiendo que a veces las semanas pueden ser complicadas y es fácil pasar por alto algunos detalles.

Respecto a la organización de los archivos, es importante que cada una gestione la ubicación correcta de sus entregas para mantener el orden y la estructura del repositorio, que es esencial para el trabajo en equipo y proyectos a gran escala. Te pediría que traslades los archivos a la carpeta correspondiente y realices otro Pull Request. Esto es parte del aprendizaje y es importante para ayudarte a familiarizarte con prácticas comunes en la gestión de proyectos de software.

Si necesitas ayuda con cómo hacerlo te dejo algunos pasos:

-Crea una nueva carpeta en el directorio correcto.
-Move los archivos a esa carpeta.
-Realiza un nuevo commit con estos cambios.
-Abre un nuevo Pull Request con estos ajustes.

¡Saludos!
Lisa

CaroSerrano committed:
chore: :truck: Creacion de carpeta alumna

Reubico los archivos subidos primeramente en la carpeta general a una carpeta específica de la clase 5: "PaulaCarolinaSerrano"
